### DISCUSSION TITLE
[功能请求]后置加载视频页右侧推荐（接下来播放）

### DISCUSSION THREAD

**Discussion by MaxHasBeenUsed** [the1812/Bilibili-Evolved#4273] [功能建议 / Ideas]
有点类似 https://github.com/the1812/Bilibili-Evolved/discussions/2897#discussion-3804900 ，但主要需求是**调整整个视频页的加载顺序**，让`自定义顶栏（customNavbar）`、`播放器置顶（新）（playerOnTopNew）`等样式更快加载。

目前看来启用以上两个样式后大致的加载顺序是：视频、视频控件、右侧（UP 主头像、视频推荐）、评论区、`自定义顶栏（customNavbar）`、`播放器置顶（新）（playerOnTopNew）`。

尝试了一下整个屏蔽掉右侧视频推荐（保留 UP 主那一栏），脚本样式的加载速度得到了提升。所以如果能后置右侧推荐，或者评论区的加载，应该对性能是有帮助的。

Firefox 上有个类似的东西 https://github.com/garywill/bslow ，不知道有没有其他共通的解决办法。

**Comment by the1812** [comment]
修改 b 站的请求加载顺序需要浏览器插件, 脚本做不了.
看起来你是希望播放器置顶能快一些, 可以参考下 https://github.com/the1812/Bilibili-Evolved/pull/4191#issuecomment-1610695402

**Reply by MaxHasBeenUsed** [reply]
谢谢！做不了就没办法了
